eventmesh-protocol-plugin/eventmesh-protocol-http/src/main/java/org/apache/eventmesh/protocol/http/resolver/HttpRequestProtocolResolver.java

```diff
@@ -31,6 +31,7 @@
 import java.util.Locale;
 import java.util.Map;
 import java.util.UUID;
+import java.util.Objects;
```

Comment:
This import should be placed here:
![image](https://user-images.githubusercontent.com/91315508/236715876-ccefee19-66e6-4625-82ee-c8c2ee50defc.png)